### DISCUSSION TITLE
超过2小时的bilibili视频无法下载，选择各种下载方式均是无法使用

### DISCUSSION THREAD

**Discussion by ForTheHumanBeing** [the1812/Bilibili-Evolved#4368] [提问 / Q&A]
浏览器：火狐 102.4.0esr (64 位)
测试bilibili视频网址：https://www.bilibili.com/video/BV1mL411z7Kf/
操作系统：uos os
症状描述：下载其他视频都正常，但发现貌似超过一定时常的下载不了，toast和strram都试了，360/703/1080p等也都试了，都不行，没有反应。期待得到解决

**Comment by hexadecimal233** [comment]
我这里可以下载啊，会不会是账号被风控了

**Reply by ForTheHumanBeing** [reply]
账号被风控的意思是?我这登录都正常,是因为下载的多 (也就下载了5-6个视频) 所以被bili官方禁止了?

**Reply by ForTheHumanBeing** [reply]
或者如果被风控了,该如何处理?
我用其他的插件试了下,貌似还是可以下载的.但就是需要改名字的那种,略显麻烦.

**Reply by ForTheHumanBeing** [reply]
![图片](https://github.com/the1812/Bilibili-Evolved/assets/141887903/57770b31-eb7a-4155-8825-7b6ae28e0bc2)
同时发现这种多P的也显示异常
